clients/drcachesim/analyzer.cpp

```diff
@@ -136,6 +136,7 @@ analyzer_tmpl_t<RecordType, ReaderType>::analyzer_tmpl_t()
 template <typename RecordType, typename ReaderType>
```

Comment:
Should also add a test that exercises the new option.

Reply:
Added a test of basic_counts with only_thread on the checked-in threadsig trace.